Neven/style.css

```diff
@@ -0,0 +1,115 @@
+*{
+    margin:0px;
+    padding:0px;  
+    box-sizing:border-box;
+  }
+  body{
+    width:100%;
+    height:100%;
+  }
+  .aside {
+    height: 100%;
+    width: 20%;
+    position: fixed;
+    z-index:1;
+    top: 0;
+    left: 0;
+    background-color: #2F4F4F;
+    color:white;
+  }
+  .aside-text{
+   text-align: right;
+   margin-top: 40%;
+   margin-right: 15px;;
+  }
+  .aside p{
+    color:#778899;
+    padding-top:10px;
+  }
+  .aside p a{
+    border:2px solid #778899;
+    text-decoration:none;
+    color: white;
+    margin-right:5px;
+    padding:5px;
+    font-size:10;
+    
+  }
+  main {
+    width: 75%;
+    background: #fff;
+    margin: 1% 22%;
+  }
+  
+  .para .avatar{
+  float: right;
+  width:50px;
+  margin-top: -30px;
+  }
+  h2{
+    padding-bottom: 4px;
+  }
+  .btn{
+    color:white;
+    padding:5px;
+  }
+  .grey{
+    color:#A9A9A9;
+  }
+  .red{
+    background-color: red;
+    color:white;
+    
+  }
+  .gray{
+    background-color: gray;
+    color: white;
+  }
+  .green{
+    background-color: green;
+    color:white;
+  }
+  .blue{
+    background-color: blue;
+    color: white;
+  }
+  .para p {
+    color: #696969;
+  }
+  .para p a{
+    text-decoration: none;
+  }
+  .para h2{
+    padding-top:20px;
+  }
+  .pic-width{
+    width:400px;
+    height: 300px;
+    margin-top:10px;
+    margin-right: 20px;
+  }
+  .info{
+    padding-top: 20px;
+    padding-bottom:20px;
+  }
+  .footer{
+    margin-top: 5%;;
+    margin-left: 28%;
+  }
+  
+  
+  
+  
```

Comment:
Don't leave too many empty lines and the end.
also, try to add one empty line between each rule.

```
  .info{
    padding-top: 20px;
    padding-bottom:20px;
  }

  .footer{
    margin-top: 5%;;
    margin-left: 28%;
  }
```